packages/webapp/src/utils/constants.js

```diff
@@ -492,6 +492,14 @@ const minStartDates = {
   MANAGEMENT_MONITORING_MIN_START_DATE
 }
 
+const ROUTES_TO_BE_DISABLED = {
```

Comment:
I think this is unnecessary, the code is already hardcoding which routes can be disabled in the helpers file when pulled through